tutorials/models/1_gnn/4_rgcn.py

```diff
@@ -294,7 +294,7 @@ def forward(self, g):
 # configurations
 n_hidden = 16 # number of hidden units
 n_bases = -1 # use number of relations as number of bases
-n_hidden_layers = 0 # use 1 input layer, 1 output layer, no hidden layer
+n_hidden_layers = 1 # use 1 input layer, 1 output layer, 1 hidden layer

```

Comment:
I think in author's rgcn, there are only two layers: one from input to hidden, the other from hidden to output. There is no hidden to hidden transformation.

Reply:
Well, this change is only used for revealing the bug, it's totally ok to follow the author's settings. I've changed that.

Reply:
Or if you think it would be more intuitive, you can also leave line 297 to have value 1, and change the line 235 to
`for _ in range(self.num_hidden_layers - 1):`